HLTrigger/Configuration/python/Eras.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+# import the relevant eras from Configuration.Eras.*
+from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
+
+def modifyHLTforEras(fragment):
+    """load all Eras-based customisations for the HLT configuration"""
+
+    # modify the HLT configuration for the Phase I pixel geometry
+    from HLTrigger.Configuration.customizeHLTTrackingForPhaseI2017 import customizeHLTPhaseIPixelGeom
+    trackingPhase1.toModify(fragment, customizeHLTPhaseIPixelGeom)
+
+    # modify the HLT configuration to run the Phase I tracking in the particle flow sequence
+    from HLTrigger.Configuration.customizeHLTTrackingForPhaseI2017 import customizeHLTForPFTrackingPhaseI2017
+    trackingPhase1.toModify(fragment, customizeHLTForPFTrackingPhaseI2017)
```

Comment:
I'd say it depends on what exactly you want to do (or how dependent of the offline tracking configuration you want to be).

If you want to run `customizeHLTForPFTrackingPhaseI2017` always when "phase1 pixel" is enabled, use `phase1Pixel`. Would it be sufficient, or do you want to run the customize only "sometimes" for phase1? (as is the case with offline tracking)